build.gradle

```diff
@@ -124,6 +129,19 @@ jar {
     from sourceSets.generated.output
 }
 
+liquibase {
+    activities {
+        main {
+            changeLogFile 'src/main/resources/db/changelog.xml'
+            url 'jdbc:postgresql://127.0.0.1:5432/jadedb'
+            username 'jade'
+            password 'jadepasswd'
```

Comment:
Seems fine to have password in here now for dev. I don't think we will want a single password for all instances of the repository metadata in all repos. What do you think?

Maybe we make a deployment story about that in the backlog?

Reply:
yes, i think maybe we get these values from vault? i'll make a story